ivy/functional/frontends/torch/tensor.py

```diff
@@ -1067,3 +1067,9 @@ def addcdiv(self, tensor1, tensor2, *, value=1):
     @with_unsupported_dtypes({"1.11.0 and below": sign_decorator_dtypes}, "torch")
     def sign(self):
         return torch_frontend.sign(self._ivy_array)
+
+    @with_unsupported_dtypes({"1.11.0 and below": ("float16",)}, "torch")
```

Comment:
This decorator should be moved to the `torch_frontend.std` frontend function not this instance method.

Reply:
hey @hirwa-nshuti  Thank you for the review. I worked on the changes you requeted and also worked on the conflicts

Reply:
Now there are other changes that are not intended to would you revert them

Reply:
Hey @hirwa-nshuti , the other changes i made were to resolve conflicts. Are these the changes you mean